client/my-sites/site-settings/form-seo.jsx

```diff
@@ -0,0 +1,257 @@
+						primary={ true }
+
+						type="submit"
+						disabled={ isDisabled || this.state.submittingForm }>
+							{ this.state.submittingForm
+								? this.translate( 'Saving…' )
+								: this.translate( 'Save Settings' )
+							}
+					</Button>
+				</SectionHeader>
+				<Card>
+					<p>
+						{ this.translate(
+							'{{b}}WordPress.com has great SEO{{/b}} out of the box. All of our themes are optimized for search engines, ' +
+							'so you don\'t have to do anything extra. However, you can tweak these settings if you\'d like more advanced control. ' +
+							'Read more about what you can do to {{a}}optimize your site\'s SEO.{{/a}}',
+							{
+								components: {
+									a: <a href={ 'https://en.blog.wordpress.com/2013/03/22/seo-on-wordpress-com/' } />,
+									b: <strong />
+								}
+							}
+						) }
+					</p>
+					<form onChange={ this.markChanged } className="seo-form">
+						<FormFieldset>
+							<FormFieldset className="has-divider is-top-only">
+								<FormLabel htmlFor="seo_meta_description">{ this.translate( 'Front Page Meta Description' ) }</FormLabel>
+								<FormInput
+									name="seo_meta_description"
+									type="text"
+									id="seo_meta_description"
+									value={ seo_meta_description }
+									disabled={ isDisabled }
+									isError={ isMetaError }
+									onChange={ this.handleMetaChange } />
+								{ isMetaError &&
+									<FormInputValidation isError={ true } text={ this.translate( 'Description can\'t be longer than 160 characters.' ) } />
+								}
+								<FormSettingExplanation>
+									{ this.translate( 'Craft a description of your site in 160 characters or less. This description can be used in search engine results for your site\'s Front Page.' ) }
+								</FormSettingExplanation>
+							</FormFieldset>
+
+							<FormFieldset className="has-divider">
+								<FormLabel htmlFor="seo_sitemap">{ this.translate( 'XML Sitemap' ) }</FormLabel>
+								<ExternalLink icon={ true } href={ sitemapUrl } target={ "_blank" }>{ sitemapUrl }</ExternalLink>
+								<FormSettingExplanation>
+									{ this.translate( 'Your site\'s sitemap is automatically sent to all major search engines for indexing.' ) }
+								</FormSettingExplanation>
+							</FormFieldset>
+
+							<FormFieldset>
+								<FormLabel htmlFor="verification_code_google">{ this.translate( 'Site Verification Services' ) }</FormLabel>
+									<p>
+										{ this.translate( 'Note that {{b}}verifying your site with these services is not necessary{{/b}} in order' +
+										' for your site to be indexed by search engines. To verify your site with a service, paste the HTML Tag' +
+										' code below. {{support}}Full instructions{{/support}}. Supported verification services:' +
+										' {{google}}Google Search Console{{/google}}, {{bing}}Bing Webmaster Center{{/bing}},' +
+										' {{pinterest}}Pinterest Site Verification{{/pinterest}}, and {{yandex}}Yandex.Webmaster{{/yandex}}.',
```

Comment:
It might be nice to line up the quotes on the same indentation level…

```js
{ this.translate(
	'This is the first' +
	'line of text and' +
	'this is the third.'
) }
```

…something like that.